inc/require_login/namespace.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/**
+ * Addons for Require Login.
+ *
+ * Pass query arg like: foo/bar?altis-auth=TOKEN
+ */
+
+namespace Altis\Security\Require_Login;
+
+use Altis;
+
+const QUERY_ARG = 'altis-auth';
```

Comment:
This should ideally be passed through the Authorization header rather than as a query parameter, as the query parameter will end up in server logs/etc. Do the tools this was created for support this?

Reply:
Not sure, @mattheu ?